src/components/TransactionsTab/TransactionsTab.tsx

```diff
@@ -229,7 +300,25 @@ const TransactionsTab = ({
                   {t("wallet.completedTransactions").toUpperCase()}
                 </LegendLine>
               </Legend>
+              <IconBinContainer
```

Comment:
Make a ClearTransactionsSelector component and throw all the components in there.

Reply:
I made a new component. I wasn't able to include `IconBinContainer` because that lives inline within `LegendContainer` in TransactionTab.tsx